ortho_seq_code/cli.py

```diff
@@ -1,6 +1,10 @@
 import click
```

Comment:
add the ~$.* to .gitingnore and remove ~$Sidhu.xlsx 

also a comment on the naming of these files, it could be something like demo.xlsx unless Sidhu means and points to some specific dataset

Reply:
Sidhu.xlsx is a sample dataset used for the tutorial and .xlsx tests, and it points to a specific research paper.